images/base/Dockerfile

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# runc (v1.0.0-rc93-pre)
```

Comment:
this change should be unecessary after https://github.com/kubernetes-sigs/kind/pull/2057 (latest containerd build should have rc93)

Reply:
seems to have caused regression :( https://github.com/kubernetes-sigs/kind/issues/2058

Reply:
The issue #2058 is now fixed.